libobs/obs.c

```diff
@@ -2123,7 +2123,14 @@ obs_source_t *obs_get_transition_by_name(const char *name)
 
 obs_source_t *obs_get_transition_by_uuid(const char *uuid)
 {
-	return obs_get_source_by_uuid(uuid);
+	obs_source_t *source = obs_get_source_by_uuid(uuid);
+
+	if (source && source->info.type == OBS_SOURCE_TYPE_TRANSITION)
+		return source;
+	else
+		obs_source_release(source);
```

Comment:
This could try to call `obs_source_release` with a NULL source. While this is protected in the API, I'd prefer we avoid passing parameters which we know will fail in the first place.

Reply:
Should by fixed now.